internal/providers/terraform/aws/tags.go

```diff
@@ -105,16 +105,27 @@ func ParseTags(defaultTags *map[string]string, r *schema.ResourceData, config Ta
 
 	tags := make(map[string]string)
 
+	// external tags (e.g. yor)
```

Comment:
is this the right precedence for yor tags? they will be "overwritten" by default tags and any explicitly defined tags?

Reply:
Yeah, I think this makes sense for the first customer use-case - we effectively only fall back to Yor tags if the code doesn't already specify them.

Reply:
but is that how terraform applies them? I would have guessed yor puts tags directly on resources so they would take precedence over default tags.

Reply:
Yeah I think that's fair. After testing yor locally, it seems like the precedence is actually: yor tags, resource tags, default tags. Since the yor "default" tags actually overwrite any resource tags I have.

Reply:
Yor overrides even regular tags - when it runs it rewrites the .tf files with the specified tag values. This is something we could do in future, but Yor gets a lot more complicated with conditionals etc., so for now we wanted to simply support taking values from the config to fall back to, regardless of conditionals etc.

But yes, it would be nice to fully support this in future, where we override defaults and tags with the Yor ones, and ideally only when conditionals match too. Yor is written in Go so we could probably just call it when a config is specified and run it "for real" once we have a Go runner.

Reply:
(sorry, wrote that before your comment appeared Alistair)

Reply:
Writing a PR to take Yor over the other tags now 👍

Reply:
I see your point -- if we're not actually supporting yor fully it seems safer to let other tags take precedence. I'm ok if you want to keep it as is, maybe just add the above as a comment explaining why they are the lowest precedence